R/plot_temp.R

```diff
@@ -18,16 +18,15 @@
 #'Jordan S. Read, Luke A. Winslow
 #'@examples 
 #'nc_file <- system.file("extdata", "output.nc", package = "glmtools")
-#'plot_temp(nc_file = nc_file, fig_path = FALSE)
-#'plot_temp(nc_file = nc_file, fig_path = 'test_figure.png', height = 3, reference = 'surface')
+#'plot_temp(nc_file = nc_file, fig_path = NULL)
 #'@export
 plot_temp <- function(nc_file='output.nc',fig_path = NULL, reference='surface', legend.title = NULL, 
                       interval = 0.5, text.size = 12, show.legend = TRUE, 
                       legend.position = 'right', plot.title = NULL){
   
   .Deprecated('plot_var',msg = 'Deprecated. Use `plot_var`, where default is var_name = `temp`')
   
-  plot_var_nc(nc_file, var_name = 'temp', fig_path = NULL, reference, legend.title, 
+  plot_var_nc(nc_file, var_name = 'temp', fig_path, reference, legend.title, 
```

Comment:
change `fig_path` to `fig_path = fig_path` and likewise for the other args that are passed from the `plot_temp` inputs? that way we can be more explicit

Reply:
Can do. Lazy coding I guess.